### ISSUE TITLE
Final hackathon ontology deliverable

### ISSUE THREAD

**Issue by cmutel** [BONSAMURAIS/BONSAI-ontology-RDF-framework#1]
- [x] Create namespace for the part of the ontology that is BONSAI-specific (i.e., which does not reside elsewhere): [purl.org](http://purl.org/) does not work for now, so we need to create something temporary 
- [x] Provide RDF schema
- [ ] Add OWL restrictions and inference rules
- [x] Create a "proper" ontology with a graphical representation. 
- [x] Coordinate classifications for activities and flow-objects (including biosphere flows) with [correspondence tables group](https://github.com/BONSAMURAIS/Correspondence-tables)
- [x] Place final ontology and RDF schema on this or new repo 
- [x] Link from wiki to github repo and provide also here any arguments for choices made or alternatives considered
- [x] Provide schema for Flow Properties  (e.g., balanceable properties)
- [x] Provide schema for property relation (a.k.a reference unit)

**Comment by kuzeko** [comment]
We can test [Widoco](https://github.com/dgarijo/Widoco) to produce the documentation of the ontology and of the RDF data.

Instructions/features: http://dgarijo.github.io/Widoco/doc/tutorial/

**Comment by cmutel** [comment]
That sounds great, but we will still need a beginner-friendly introduction to the core ontology concepts as part of the bonsai website.

**Comment by kuzeko** [comment]
I agree in general, but I need to understand the use case so that I can go beyond what is already in the [README](https://github.com/BONSAMURAIS/BONSAI-ontology-RDF-framework) and provide effective instructions.

For who is this introduction written? What are they trying to accomplish?
